### ISSUE TITLE
vscode.dev: better focus for the git password entry field when git pushing from terminal

### ISSUE THREAD

**Issue by kashav** [microsoft/vscode#216913]
I ran git push from a terminal in a hosted instance of vscode.dev and didn't notice that it was opening an "enter password" input. It seemed like the git push was just hanging. I was using the default light theme but don't easily notice the input field in the dark theme either. I've attached a screenshot below.

Input element: `<div class="quick-input-header"></div>`:

<img width="686" alt="Screenshot 2024-06-23 at 13 01 45" src="https://github.com/microsoft/vscode/assets/8116382/67e2c1a9-31eb-45e0-9749-09027637331e">

Version: 1.90.2
Commit: 5437499feb04f7a586f677b155b039bc2b3669eb
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
Embedder: vscode.dev

**Comment by kashav** [comment]
I guess playing around with the shadow here might make things better: https://github.com/microsoft/vscode/blob/524d1ecedd33104381e7e0a6715c13813128f89c/src/vs/platform/theme/common/colors/editorColors.ts#L331-L333.

**Comment by lszomoru** [comment]
//cc @daviddossett, @hbons

**Comment by daviddossett** [comment]
Looks like the input isn't focused which isn't helping here. I think I've seen this in one or two other issues cc @TylerLeonhardt

**Comment by kashav** [comment]
I believe the input was focused but it lost focus because of how I took the screenshot. This is to say that there was a browser outline around the "Password" input when it had focus, but the outline wasn't noticeable and the focus didn't draw my attention because of the colour contrast (IMO).

**Comment by daviddossett** [comment]
Is this Light Modern/Dark Modern? I'll explore some shadow and/or border adjustments. The challenge here is that some changes to tiny quick input instances like this can sometimes feel like too much in more typical cases (e.g. the command palette, file picker, etc.).

**Comment by kashav** [comment]
Yeah, this is Light +. I wonder if there's any other way to shift attention when a modal of any kind is opened?

**Comment by lszomoru** [comment]
Adjusting label and ownership.

**Comment by TylerLeonhardt** [comment]
This does feel more like a theme issue than a quick pick issue.
